scip.proto

```diff
@@ -85,7 +85,7 @@ message Document {
   string relative_path = 1;
   // Occurrences that appear in this file.
   repeated Occurrence occurrences = 2;
-  // Symbols that are defined within this document.
+  // Non-local symbols that are defined within this document.
```

Comment:
Where should local symbols have `SymbolInformation`? This comment somewhat implies that this list must contain only non-local symbols.

Reply:
`SymbolInformation` should be optional for all symbols (local or not) after we merged this PR here https://github.com/sourcegraph/scip/pull/36

Before that PR, it was necessary to emit `SymbolInformation` for every symbol that has a definition occurrence.

Reply:
I guess maybe I should reword it. The point I was trying to make was, that it isn't useful to have information about local symbols in this list (purely information in the occurrences list should be fine). Is that correct or am I misunderstanding the purpose of this list?